Public/Src/Utilities/Native/IO/FileUtilities.cs

```diff
@@ -560,8 +560,8 @@ public static unsafe FileAttributes GetFileAttributesByHandle(SafeFileHandle fil
             }
         }
 
-        /// <see cref="IFileSystem.GetFileAttributes(string)"/>
-        public static FileAttributes GetFileAttributes(string path) => s_fileSystem.GetFileAttributes(path);
+        /// <see cref="IFileSystem.GetFileAttributes(string, bool)"/>
+        public static FileAttributes GetFileAttributes(string path, bool throwOnFailure = true) => s_fileSystem.GetFileAttributes(path, throwOnFailure);

```

Comment:
>bool throwOnFailure = true [](start = 68, length = 26)

Having this `bool throwOnFailure` looks icky. Just like we have `DeleteFile` and `TryDeleteFile`, why don't we add `TryGetFileAttributes` ?